2017-bae-europe/eu_future/buff_but_sink.xml

```diff
@@ -0,0 +1,5827 @@
+
+<simulation>
+  <control>
+    <duration>960</duration>
+    <startmonth>1 </startmonth>
+    <startyear>1970 </startyear>
+  </control>
+
+
+  <archetypes>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Source</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Sink</name>
+    </spec> 
+    <spec>
+      <lib>cycamore</lib>
+      <name>Reactor</name>
+    </spec>
+    <spec>
+        <lib>cycamore</lib>
+        <name>Mixer</name>
+    </spec>
+    <spec>
+      <lib>agents</lib>
+      <name>NullRegion</name>
+    </spec>
+    <spec>
+      <lib>agents</lib>
+      <name>NullInst</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Separations</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>DeployInst</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Separations</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Enrichment</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>FuelFab</name>
+    </spec>
+  </archetypes>
+
+
+<facility>
+  <!--MELOX 195tHM/y-->
+  <!-- 1995 -->
+  <!-- world-nuclear.org FRANCE-->
+    <name>MELOX</name>
+    <config>
+      <Mixer>
+        <in_streams>
+          <stream>
+            <info>
+              <mixing_ratio>.09</mixing_ratio>
+              <buf_size>16250</buf_size>
+            </info>
+            <commodities>
+              <item>
+                <commodity>uox_Pu</commodity>
+                <pref>1.0</pref>
```

Comment:
Well, the intention was to have the preferences so that someone who later would like to change things around can change them later. But for the current state, it does nothing!